app/connectors/FinancialDetailsConnector.scala

```diff
@@ -243,33 +241,28 @@ class FinancialDetailsConnector @Inject()(
       .map { response =>
         response.status match {
           case OK =>
-            Logger("application").debug(s"Status: ${response.status}, json: ${response.json}")
-            response.json.validate[OutstandingChargesModel].fold(
+            Logger("application").info(s"Status: ${response.status}, json: ${response.json}")
```

Comment:
if we can revert this back to "DEBUG" as we don't need response details to be exposed in PROD...